templates/components/form/fields_macros.html.twig

```diff
@@ -99,12 +99,17 @@
       'id': '%id%'
    }|merge(options) %}
 
-   {% set field %}
-        {% import 'components/form/basic_inputs_macros.html.twig' as _inputs %}
-        {{ _inputs.text(name, value, options) }}
-   {% endset %}
+   {% if options.readonly %}
+        {{ _self.htmlField(name, value, label, options) }}
```

Comment:
Not sure this should be changed specifically for text fields. If the issue is the border in the print, it could be solved with CSS.